src/worker/core/draft/genPlayersWithoutSaving.ts

```diff
@@ -57,12 +58,15 @@ const genPlayersWithoutSaving = async (
 	// Do one season at a time, keeping the lowest pot players in college for another season
 	let enteringDraft: typeof remaining = [];
 
-	for (let i = 0; i < 4; i++) {
-		let cutoff = 0; // For football, only juniors and seniors
+	for (let i = 0; i < draftAge[1] + 1; i++) {
+		let cutoff = draftAge[1] - draftAge[0]; // For football, only juniors and seniors by default.
 
 		if (isSport("basketball") || i >= 2) {
```

Comment:
Or... not sure if this condition makes sense at all any more, shouldn't this always run now? I think just delete the `if`.